sdk/identity/Azure.Identity/TROUBLESHOOTING.md

```diff
@@ -1,270 +1,222 @@
-﻿## Troubleshooting Azure Identity Authentication Issues
+﻿# Troubleshooting Azure Identity Authentication Issues
 
-The Azure Identity SDK offers various `TokenCredential` implementations. The most common exceptions observed for failure scenarios
-are `CredentialUnavailableException`
-and `AuthenticationFailedException`.
-
-- `CredentialUnavailableException` indicates that the credential cannot execute in the current environment setup due to lack of required configuration.
-- `AuthenticationFailedException` indicates that the credential was able to run/execute but ran into an authentication issue from the server's end. This can happen
-  due to invalid configuration/details passed in to the credential at construction time.
-
-This troubleshooting guide covers mitigation steps to resolve these exceptions thrown by various `TokenCredential` implementations in the Azure Identity .NET client
-library.
+This troubleshooting guide covers failure investigation techniques, common errors for the credential types in the Azure Identity .NET client library, and mitigation steps to resolve these errors.
 
 ## Table of contents
+- [Handling Azure Identity Exceptions](#handling-azure-identity-exceptions)
+  - [AuthenticationFailedException](#authenticationfailedexception)
+  - [CredentialUnavailableException](#credentialunavailableexception)
+- [Finding Relevant Information in Exception Messages](#finding-relevant-information-in-exception-messages)
```

Comment:
I think we should also include the `permission issues` section from the previous TSG - about giving AAD permissions to resources and errors thrown from such scenarios.

Reply:
Added